guides/15_Background_Filter_Video_Processor.md

```diff
@@ -0,0 +1,145 @@
+
+```
+Refused to connect to
+'https://static.sdkassets.chime.aws/bgblur/workers/worker.js…'
+because it violates the document's Content Security Policy.
+```
+
+If you need guidance with correct CSP configuration to use background blur, contact AWS Support.
+
+### Cross-Origin Opener Policy
+
+Some execution modes require careful configuration of web security, including less common headers like [`Cross-Origin-Opener-Policy`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy). To opt in to these execution modes, send
+
+```
+Cross-Origin-Opener-Policy: same-origin
+Cross-Origin-Embedder-Policy: require-corp
+```
+
+when serving your application HTML. These headers are optional but advised: they improve web security, as well as allowing the browser to support technologies like `SharedArrayBuffer`.
+
+## Checking for support before offering background blur
+
+Some browsers support the Amazon Chime SDK but do not support background blur. Additionally, some devices are not fast enough to keep up with real-time video while blurring the background.
+
+The SDK provides a static method to allow you to cheaply check for the required browser features:
+
+```typescript
+import { BackgroundBlurVideoFrameProcessor } from 'amazon-chime-sdk-js';
+BackgroundBlurVideoFrameProcessor.isSupported();
+```
+
+If `isSupported` returns `true`, you can instantiate the blur video frame processor.
+
+```typescript
+const processors = [];
+if (BackgroundBlurVideoFrameProcessor.isSupported()) {
+    const blurProcessor = await BackgroundBlurVideoFrameProcessor.create();
+    processors.push(blurProcessor);
+}
+```
+
+The create method will pre-load the model file and prepare background blur for use. Failure to load necessary resources will result in `isSupported` returning `false`. 
+
+
+## Adding background blur to your application
+
+Background blur integrates with the SDK by providing a new implementation of the `VideoFrameProcessor`
+
+Once you have checked for support and created a blur processor, you add the processor to the transform device.
+
+```typescript
+const processors = [];
+if (BackgroundBlurVideoFrameProcessor.isSupported()) {
+    const blurProcessor = await BackgroundBlurVideoFrameProcessor.create();
+    processors.push(blurProcessor);
+}
+let transformDevice = new DefaultVideoTransformDevice(logger, device, processors);
+```
+
+## Configuration
+
+Both `isSupported` and `create` accept _specifications_ — `BackgroundFilterSpec` structures — as input. These allow you to describe the model attributes, execution approach, and other configuration options that define how the feature should operate.
```

Comment:
Do want to clarify  the parameters for `BackgroundFilterSpec` and `BackgroundFilterOptions` here?

Reply:
Added a comment to clarify structures.